src/tools/rbd/Utils.cc

```diff
@@ -762,5 +762,93 @@ std::string timestr(time_t t) {
   return buf;
 }
 
+int export_diff(librbd::Image& image, const char *fromsnapname,
```

Comment:
I need to refer these functions in Export/Import and ExportDiff/ImportDiff. Then I put them in Utils. a new file? Such as src/tools/rbd/Diff.cc?